apps/desktop/src/components/CommitContextMenu.svelte

```diff
@@ -214,24 +194,6 @@
 						}}
 					/>
 				</ContextMenuSection>
-				<ContextMenuSection>
-					<ContextMenuItem
-						label="Create dependent branch above"
-						disabled={refCreation.current.isLoading}
-						onclick={async () => {
-							await handleCreateNewRef(ensureValue(stackId), commitId, 'Above');
```

Comment:
@PavelLaptev This hurts 😅 as it removes functionality that can't be achieved in another way right now.

Maybe from a UX perspective, we can rethink this operation so it's more of a 'split'? Could this be achievable differently? I keep thinking about the insertion-marker when creating new commits - inserting a reference at a selected point feels similar.